float32 young_modulus
---
bool success
